src/main/java/org/broadinstitute/hellbender/tools/recalibration/RecalibrationReport.java

```diff
@@ -331,6 +322,12 @@ else if (argument.equals("sort_by_all_columns"))
                 RAC.SORT_BY_ALL_COLUMNS = Boolean.parseBoolean((String) value);
         }
 
+        if (!RAC.COVARIATES.equals(new StandardCovariateList().getStandardCovariateClassNames())) {
+            throw new UserException("Non-standard covariates are not supported. Only the following are supported " + new StandardCovariateList().getStandardCovariateClassNames() + " but was " + RAC.COVARIATES);
+        }
+        if (RAC.DO_NOT_USE_STANDARD_COVARIATES){
+            throw new UserException("Non-standard covariates are not supported. Only the following are supported " + new StandardCovariateList().getStandardCovariateClassNames() + " but no_standard_covs was true");
+        }
```

Comment:
I really don't see why we still need these fields in the argument collection (and these crazy checks here) if they are just constants -- can't we refactor whatever code is relying on them to use the values we now require them to have directly? Could you explain?